### ISSUE TITLE
New SC for audio quality of canned video/audio?

### ISSUE THREAD

**Issue by detlevhfischer** [w3c/wcag#1686]
Hi,
Recently, a user enraged about the abysmal audio quality of some info video on a federal website enquired whether audio quality is part of the accessibility requirements of the German directive (which points to the EN 301 549 and via that to WCAG 2.1 and additional stuff). 
The 'additional stuff' part actually checks the bandwidth aspect of audio quality in EN 301 549  requirement "6.1 Audio bandwidth for speech", but if the audio source quality is bad I guess this is a separate problem.
It stands to reason that bad audio quality would disproportionally affect people with hearing impairments, so I think in principle such a requirement (and SC) might qualify. I am not aware, however, if there are good automated tools to check audio (speech) quality and where you would draw the line between pass and fail. I'd be curious to hear other people's opinion on that.

**Comment by patrickhlauke** [comment]
urgh...how would you be able to test it *if* it's dependent on availability, the end user's connection speed/reliability, etc? you could likely only test it in a very theoretical "this prerecorded audio is 44.1KHz/16bit at source". then look at the type of encoder used? lossy vs lossless? and what the original audio quality itself is/was? let alone testing it for live audio which may vary. i personally would shy away from making some kind of hard pass/fail determination here.

**Comment by detlevhfischer** [comment]
Well, the assessment could or should be separated from bandwidth/connection speed etc. which is the subject of EN 6.1 (but there nor for canned stuff). As for videos that are downloaded / buffers by the client (if I am not mistaken). If there was a clever tool measuring information carried in voice vs. background noise, echo etc. it might be measurable on a scale. I have no idea whether such tools exist, for example used to test sound for fitness for broadcasting. But I admit it would be hard to test - just raising it as an issue here nevertheless.

**Comment by patrickhlauke** [comment]
as a third-party auditor who also relies on accessing/testing things from an actual remote connection, this would make testing rather...non-representative. just as a gut feeling, i would be careful floating this before less technically-minded folks jump in and think yeah that's a great idea without any understanding or consideration for the practicalities of testing this.

there's already currently a big question mark over the 1.4.7 AAA SC ... about how this should actually be tested, if there are tools available, etc. would be good to clean house first on this (2.0) SC before hoping for other clever tools...

**Comment by mraccess77** [comment]
Audio CAPTCHA are notoriously bad here - on purpose but leave folks out of luck.

**Comment by patrickhlauke** [comment]
This sort of thing is likely easier to spec for a non-binary pass/fail SC that needs to provide some form of reproducible/measurable "understandability of the audio" threshold. This is really something that, if worded in a very subjective way ("make sure that audio quality is understandable" and lots of handwaving), is easier to write down - but of course will then fall victim to inconsistent scoring from different auditors. I'd almost think a W3C note, rather than a hard SC, may be more successful here initially.

**Comment by alastc** [comment]
Assuming that it was a measure of the audio quality/understandability of speech content, it sounds like a 'holistic' test in WCAG 3.0 sort of thing.

Presumably this is something that would most affect people with both hearing and visual impairments? Otherwise captions would seem to be the obvious solution.
